db/structure.sql

```diff
@@ -12,21 +12,29 @@ SET client_encoding = 'UTF8';
 SET standard_conforming_strings = on;
 SELECT pg_catalog.set_config('search_path', '', false);
 SET check_function_bodies = false;
+SET xmloption = content;
```

Comment:
This should only be making expected changes to the `structure.sql` file - ie just the `CREATE FUNCTION` block - all the extra changes here should be dropped